OutOfSchool/OutOfSchool.AuthCommon/Views/Auth/Login.cshtml

```diff
@@ -48,6 +48,10 @@
                            asp-route-returnUrl="@Model.ReturnUrl">@SharedLocalizer["Forgot password?"]</a>
                     </div>
                 </div>
+                <div class="checkbox-container">
+                    <input asp-for="RememberMe" type="checkbox" id="rememberMe">
+                    <label>@SharedLocalizer["Remember me?"]</label>
```

Comment:
don't see this value added to shared resources in this PR, maybe forgot to commit?